alacritty.yml

```diff
@@ -450,6 +468,27 @@
     # binding section.
     #modifiers: None
 
+# Regex hints
+#
+# Terminal hints can be used to find text in the visible part of the terminal
+# and pipe it to other applications.
+#hints:
+  # Keys used for the hint labels
+  #alphabet: "jfkdls;ahgurieowpq"
+
+  # List with all available hints
+  #
+  # The fields `command`, `binding.key` and `binding.mods` accept the same
+  # values as they do in the `key_bindings` section.
+  #
+  # Example:
+  #  - regex: "alacritty/alacritty#\\d*"
+  #    command: firefox
+  #    binding:
+  #      key: G
+  #      mods: Control|Shift
+  #enabled: []
```

Comment:
I've actually considered using active vs enabled, but chose to go with `enabled` because "active" sounds a bit more like they'd be always visible? Like an active hint to me is a hint that is currently being selected. So that's why I prefer enabled over active.